packages/core/upload/admin/src/pages/SettingsPage/index.js

```diff
@@ -201,7 +201,7 @@ export const SettingsPage = () => {
                           hint={formatMessage({
                             id: getTrad('settings.form.sizeOptimization.description'),
                             defaultMessage:
-                              'Enabling this option will optimize the file size without compromising on the quality.',
+                              'Enabling this option will reduce the image size and slightly reduce its quality.',
```

Comment:
```suggestion
                              'Enabling this option reduces the size of the image and might reduce its quality, depending on the type.',
```

I'd leave out the word "slightly" because it might be confusing (ex: "How much is slightly?")

Reply:
It is sure that the quality will decrease, so I would not say "might".
The quality is set to 80% but it doesn't say a lot, it's still a high quality. I think people can do their own tests to see how slight it is. What do you think ?

Reply:
The reason why I'd say "migh, depending on" is, that it won't happen to non-jpeg files as far as I understood?

Reply:
Ok I understand why you say that, actually it will happen to png files too.
The default quality value is different (80 for jpg, 100 for png) but the applied value is 80 for all.